src/main/java/racingcar/domain/CarFacility.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarFacility {
+
+    public CarFacility() {
+    }
+
+    public List<Car> buildCar(int carNum) {
+        List<Car> carList = new ArrayList<>();
```

Comment:
```suggestion
        List<Car> cars = new ArrayList<>();
```
자료구조를 포함한 네이밍보다 cars가 자연스러워 보이네요 :)